domains/rem/src/ui/EntityOptionsRow/index.tsx

```diff
@@ -24,7 +24,7 @@ export const EntityOptionsRow: React.FC<EntityOptionsRowProps> = ({
 
 	return (
 		<div className='ee-entity-option__wrapper'>
-			<div className='ee-entity-option__options'>
+			<div className={`ee-entity-option__options${hideAddNew ? '--single' : ''}`}>
```

Comment:
ooo this is not quite right...
the one catch with BEM is that you need to keep the base class when adding a modifier.
the way you have this it would switch between:

```js
className={`ee-entity-option__options`}>
```

and 

```js
className={`ee-entity-option__options--single`}>
```

but the modifier should be an additional class not a modification to the existing.

so this needs to be:

```suggestion
			<div className={`ee-entity-option__options${hideAddNew ? ' ee-entity-option__options--single' : ''}`}>
```

Reply:
I see, I'll update this.